plenum/test/common/test_digest_validation.py

```diff
@@ -0,0 +1,103 @@
+import json
+
+import pytest
+from indy.did import create_and_store_my_did
+
+from plenum.common.exceptions import RequestNackedException
+from plenum.common.request import Request
+from plenum.test.helper import sdk_gen_request, sdk_multisign_request_object, sdk_send_signed_requests, \
+    sdk_get_and_check_replies, sdk_random_request_objects
+
+from plenum.common.constants import CURRENT_PROTOCOL_VERSION, DOMAIN_LEDGER_ID
+
+from plenum.common.util import randomString
+from stp_core.loop.eventually import eventually
+
+
+@pytest.fixture(scope='function')
+def op(looper, sdk_wallet_stewards):
+    wh, did = sdk_wallet_stewards[0]
+    seed = randomString(32)
+    new_steward_did, new_steward_verkey = looper.loop.run_until_complete(
```

Comment:
Why this is a steward DID? It looks like it created a common user.

Reply:
fixed